romancal/jump/tests/test_jump_step.py

```diff
@@ -109,6 +110,7 @@ def _setup(ngroups=10, readnoise=10, nrows=20, ncols=20,
     return _setup
 
 
+@pytest.mark.xfail
```

Comment:
Can you add a reason for xfailing these tests

Reply:
Updated to skip the tests on github actions